CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ## [0.25.0] - 2020-10-27
+### Fixed
+- `ItemContextWrapper` key.
```

Comment:
you should add this under the `[Unreleased]` section, not under a previous released tag

Reply:
Fixed @lucasecdb ty